web/menu.jspf

```diff
@@ -71,24 +71,153 @@ org.opensolaris.opengrok.web.Util"
             pMap.put(name, esc);
         }
 %>
+<script type="text/javascript" src="<%= request.getContextPath() %>/js/jquery-3.2.0.min.js"></script>
 <script type="text/javascript">/* <![CDATA[ */
-    document.domReady.push(function() { domReadyMenu(); });
+    document.xrefPath = '<%= request.getContextPath() + Prefix.XREF_P
+    %>';document.domReady.push(function() { domReadyMenu(); });
 /* ]]> */
+var initialWidth = null;
+var isFirst = true;
+var checkWidth = function(){
+    var width = (window.innerWidth > 0) ? window.innerWidth : screen.width;
+	console.log(width);	
+	if(width <= 770) {
+		console.log('width less that 770');
+		//$(".fixture").each(function() {
+		//var detach = $(this).find(".fixture-date").detach();
+		//$(detach).insertAfter($(this).find(".teams"));
+		//})
+		$("#sbox").each(function() {
+		var detach = $(this).find("#qtbl").detach();
+		$(detach).insertAfter($(this).find("#ptbl"));
+		var detach1 = $(this).find("#ltbl").detach();
+		$(detach1).insertBefore($(this).find("#qtbl"));
+		});
+		console.log('width changed');
+	}else{
+		//revert back
+		$("#sbox").each(function() {
+		var detach = $(this).find("#qtbl").detach();
+		$(detach).insertBefore($(this).find("#ptbl"));
+		});	
+	}	
+	initialWidth = width;
+};
+$(document).ready(function(){ 
```

Comment:
I am thinking of moving the methods added in menu.jspf to utils-0.0.9.js. In that case, since the util file is loaded after jquery, it have no issues. Would that be ok?